tipg4/TIP.g4

```diff
@@ -64,7 +64,11 @@ statement : blockStmt
     | errorStmt
 ;
 
-assignmentStmt : expr '=' expr ';' ;
+assignmentStmt : pointerAssignment | variableAssignment ;
+
+variableAssignment : expr '=' expr ';' ;
```

Comment:
I would use varExpr here for the LHS of a variableAssignment

Reply:
got it, i'll make the change